build-system/common/ci.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Provides functions that extract various kinds of CI state.
```

Comment:
I assume the long-term plan is to replace all of `travis.js` with "CI-agnostic" functions in this file?

Reply:
Precisely. We will still retain a subset of provider specific files like `travis.js` / `github-actions.js` and add new ones, but build system code will eventually use the CI-agnostic versions in `ci.js`.